google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineTask.java

```diff
@@ -21,12 +21,20 @@
 /**
  * Encapsulates a task to be run on Google App Engine.
  */
-public interface AppEngineTask {
+public abstract class AppEngineTask {
```

Comment:
Yea for sure. The other approach here would be to keep it an interface so that the onCancel method is very explicitly published, and then live with empty implementations of it.

But if we don't feel strongly in this case, I'll leave it as is.